src/d3a/models/market/market_structures.py

```diff
@@ -20,20 +20,22 @@
 
 
 class Offer:
-    def __init__(self, id, price, energy, seller, original_offer_price=None):
+    def __init__(self, id, price, energy, seller, original_offer_price=None, energy_origin=None):
```

Comment:
As discussed briefly offline. How about renaming the `energy_origin` to `seller_origin` in the offer case and `buyer_origin` the bid case. This would also be consistent with the parameter naming in the Trade object. 
@spyrostz What do you think?

Reply:
@hannesdiedrich @faizan2590 I am not sure, but I think I would prefer Hannes's suggestion too. The reason for that is that the for the bid case this name is misleading - there is no energy produced for the bid, therefore energy_origin looks weird for this case. If it is desirable to keep the same name for both bid and offer, I would suggest renaming both to something like 'origin', 'source_area' for instance. If this is not a problem, then Hannes's suggestion looks good to me.